content/get-involved.html

```diff
@@ -8,27 +8,31 @@
 <h1>Get involved</h1>
 
 <p>
-There are various ways how you can get involved including contributions on GitHub <i class="fa fa-github "></i>: <br />
-<a href="https://github.com/eclipse-thingweb/node-wot/">https://github.com/eclipse-thingweb/node-wot/</a>
+There are various ways to get involved via GitHub <i class="fa fa-github "></i>:
+<a href="https://github.com/eclipse-thingweb">https://github.com/eclipse-thingweb</a>
+
+</p>
+<p>
+You can pick one of the Eclipse Thingweb components such as node-wot or playground and start using them.
+Even better, you can contribute to their source code!
+Below are some examples of what you can contribute with:
 </p>
 <ul>
-  <li>Create and contribute your <i>own</i> protocols bindings besides HTTP, CoAP, Websockets, ...</li>
-  <li>Build your WoT solutions and share them</li>
-  <li>Report any issue you encounter (and even better provide already a fix)</li>
+  <li>node-wot: Create and contribute your <i>own</i> protocols bindings besides HTTP, CoAP, Websockets, ...</li>
+  <li>playground: Add tools that take Thing Descriptions and generate developer tooling for other systems</li>
+  <li>All components: Report any issue you encounter (and even better, provide a fix)</li>
```

Comment:
We should fix the css, list items does not use the same font and color of `p` blocks:
![image](https://github.com/eclipse-thingweb/website/assets/3170227/d58527fd-acbe-4b90-9e6c-4a76e835f9f4)

Reply:
fixed by #59